### PR TITLE
Mastermind game part

### PR DESCRIPTION
**Pull request by d-beloved**
@mradeybee sir, I haven't finished the code, I just wanted to show you this in order to know if I'm on track and also to test my use of gitHub

### CODE REVIEW

File: masterMind.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	returns it to the caller*/ 
+}
+
+/*function to compare the player's guess
+compared with the computer's random pick, loops 12 times which is the number of try the player's got*/
+function compareGuess() {
+	var i = 0
+	for (i; i <= 11; i++) {
+
+		compareGuess = 0;
```
Comment (mradeybee) [inline masterMind.js:23]: Check your spacing here. Use 2 spaces as one tab.

File: masterMind.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+alert("welcome to MASTERMIND");
+var choice = prompt("Would you like to (p)lay, read the (i)nstructions, or (q)uit?").toLowerCase();
+
+if (choice === "p" || choice === "play") {
+	
+	playGame();
+
+	alert("I have generated a beginner sequence with four elements made up of: (r)ed,
```
Comment (mradeybee) [inline masterMind.js]: use console.log, instead of alert

**Review by mradeybee — CHANGES_REQUESTED** [review]
Use single tab spacing all through, set your tab to 2 spaces. It will give you a cleaner code. Also, this is a CLI game, so use `console.log` when you want to print out anything.

**Comment by d-beloved** [comment]
Thanks sir, I have effected the changes....
coming up with more features on the game.

**Pull request merged**